src/main/java/software/bernie/techarium/display/container/component/WhiteListItemSlot.java

```diff
@@ -0,0 +1,25 @@
+package software.bernie.techarium.display.container.component;
+
+import java.util.List;
+
+import net.minecraft.inventory.IInventory;
+import net.minecraft.inventory.container.Slot;
+import net.minecraft.item.ItemStack;
+import software.bernie.techarium.helper.ItemsHelper;
+
+public class WhiteListItemSlot extends Slot{
+
+	List<ItemStack> stacks;
+	
+	public WhiteListItemSlot(IInventory inv, int id, int posX, int posY, List<ItemStack> whitelist) {
```

Comment:
```suggestion
	public WhiteListItemSlot(IInventory inv, int id, int posX, int posY, ItemStack... whitelist) {
```
Make whitelist a varargs paramer